### PR TITLE
Wwdc 2015 meetup

### PR DESCRIPTION
**Pull request by agough**
I've had a go at a post for WWDC 2015 keynote live stream, feel free to edit as you see fit. I've also setup the GetInvited page (it's currently in draft pending this PR being merged, but link should be good).

@mauricerkelly / @marramgrass can you guys confirm ShopKeep address I thought you guys were moving, but Google still shows you at your original address.

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -53,13 +53,13 @@ <h3 id="countdown">A gathering of Belfast cocoaheads</h3>
           
           <!-- Next meetup details starts  -->
           
-          <p>Next meetup: December 15th 2014 at 6:00pm</p>
+          <p>Next meetup: June 8th 2015 at 6:00pm</p>
```
Comment (mauricerkelly) [inline _layouts/default.html]: We should maybe say 5:45pm - keynote is at 6pm so would be good to get people in and settled.

**Comment by mauricerkelly** [comment]
There’s a section below in the default layout file titled “Our Next Event” which has another link to the last event. If you haven’t got the time to update it, let me know and I’ll do it.

**Comment by mauricerkelly** [comment]
Otherwise, looks good!

**Comment by agough** [comment]
That's the widget updated and the Get Invited event is now published. So merge when ready. Thanks.

**Comment by mauricerkelly** [comment]
LGTM!

**Pull request merged**